src/mpi/coll/include/coll_impl.h

```diff
@@ -87,17 +87,17 @@ int MPII_Coll_finalize(void);
         if (mpi_errno)                                                  \
             MPIR_ERR_POP(mpi_errno);                                    \
                                                                         \
-        mpi_errno = MPIR_Sched_create(&s);                              \
+        mpi_errno = MPIR_Sched_create(&s, MPIR_SCHED_KIND_REGULAR);     \
         if (mpi_errno)                                                  \
             MPIR_ERR_POP(mpi_errno);                                    \
                                                                         \
         mpi_errno = fn(__VA_ARGS__, comm_ptr, s);                       \
         if (mpi_errno)                                                  \
             MPIR_ERR_POP(mpi_errno);                                    \
                                                                         \
-        mpi_errno = MPIR_Sched_start(&s, comm_ptr, tag, request);       \
-        if (mpi_errno)                                                  \
-            MPIR_ERR_POP(mpi_errno);                                    \
+            mpi_errno = MPIR_Sched_start(s, comm_ptr, tag, request);    \
+            MPIR_ERR_CHECK(mpi_errno);                                  \
+        }                                                               \
```

Comment:
the `}` here seems to be incorrect.  The code indentation also need fixes.

Reply:
fixed.